docs/content/node-operation/index.mdx

```diff
@@ -1,21 +1,42 @@
 ---
-title: Hello Node Operator!
+title: Flow Node
 sidebar_title: Introduction
-description: How to run your own Flow node
+description: Introduction Flow node
 ---
 
-To run a node in the Flow network, the node must:
-* be registered with sufficient stake
-* be authorized by the service account
+The Flow node is the vital component that supports the network and enables you to participate in the Flow decentralized blockchain.
+A blockchain network is a collection of interconnected nodes working together to validate transactions and blocks using consensus.
+Therefore, you are contributing to the network as a node operator by running a Flow node.
 
-Before proceeding, ensure you have the stake required for your new node and that your node will be authorized by the service account (apply [here](https://www.onflow.org/node-validators)).
+## What are nodes
+Blockchain nodes are network contributors, and as such, their device is authorized 
+to maintain the distributed ledger and act as 
+a central point of contact for a variety of network operations.
+In addition, a blockchain node's primary responsibility is verifying the 
+legitimacy of each new block of network transactions. 
 
-To set up a new Flow node you will need to complete the following steps:
+## Functions of nodes
+A block broadcast is sent to all network nodes when a node operator tries adding to the blockchain a new block of transactions. However, a node can accept or reject a block depending on its validity (validity of signatures and transactions). Nodes accomplish the following in essence:
```

Comment:
This seems to be a very generic description of a blockchain node.
Lets pull content from here https://docs.onflow.org/node-operation/node-roles/ if you are trying to give a generic overview of a node.

Reply:
Thanks for the input.

Reply:
fixed